src/main/java/baseball/Balls.java

```diff
@@ -0,0 +1,44 @@
+package baseball;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Balls {
+    private List<Ball> list;
```

Comment:
자료구조의 이름을 그대로 변수의 이름으로 가져가는 것은 지양해야합니다.

만약 요구사항이 바뀌어 List가 아닌 Set으로 바뀌어야한다면 어떨까요?

아래와 같이 values 또는 balls를 추천드립니다 :)
```suggestion
    private List<Ball> values;
```